src/ModelContextProtocol.Core/Server/AIFunctionMcpServerTool.cs

```diff
@@ -376,15 +377,35 @@ internal static IReadOnlyList<object> CreateMetadata(MethodInfo method)
         return meta;
     }
 
-    /// <summary>Regex that flags runs of characters other than ASCII digits or letters.</summary>
 #if NET
```

Comment:
I wish we had an OOB version of the regex generator so that all this duplication could be avoided.